tests/test_general_linear_group.py

```diff
@@ -2,11 +2,8 @@
 Unit tests for General Linear group.
```

Comment:
You took test_general_linear_group_tensorflow as baseline here, instead of test_general_linear_group_numpy. Aren't you missing some tests?